src/assets/v1.3.0/simulate_tx_accessor.json

```diff
@@ -144,6 +145,7 @@
     "54211": "0x727a77a074D1E6c4530e814F89E618a3298FC044",
     "56288": "0x727a77a074D1E6c4530e814F89E618a3298FC044",
     "57000": "0x727a77a074D1E6c4530e814F89E618a3298FC044",
+    "58008": "0x59AD6735bCd8152B84860Cb256dD9e96b85F69Da",
```

Comment:
Contract is not verified, but was deployed: https://explorer.sepolia.publicgoods.network/address/0x59AD6735bCd8152B84860Cb256dD9e96b85F69Da?tab=contract